src/processors.js

```diff
@@ -277,6 +358,15 @@ class WhisperFeatureExtractor extends FeatureExtractor {
         return padded;
     }
 
+    /**
+     * Calculates the complex Short-Time Fourier Transform (STFT) of the given framed signal.
+     * 
+     * @param {Array<Array<number>>} frames - A 2D array representing the signal frames.
+     * @param {Array<number>} window - A 1D array representing the window to be applied to the frames.
```

Comment:
Currently we use two styles, `Array<number>` vs `number[]`, should we just stick to one?
```suggestion
     * @param {number[][]} frames - A 2D array representing the signal frames.
     * @param {number[]} window - A 1D array representing the window to be applied to the frames.
```